package.json

```diff
@@ -50,7 +50,9 @@
     "version:drive:androidbuild": "perl -pi -e 's,android-versionCode=\"[0-9]+\",android-versionCode=\"'`echo ${npm_package_version} | tr -d .`'0000\",g' ./src/drive/targets/mobile/config.xml",
     "version:drive:useragent": "replace 'value=\"(.+)\\d+\\.\\d+\\.\\d+\"' 'value=\"$1'$npm_package_version'\"' ./src/drive/targets/mobile/config.xml",
     "version:drive": "npm-run-all --parallel 'version:drive:*'",
-    "version:photos": "replace '\\d+\\.\\d+\\.\\d+' $npm_package_version ./src/photos/targets/manifest.webapp"
+    "version:photos": "replace '\\d+\\.\\d+\\.\\d+' $npm_package_version ./src/photos/targets/manifest.webapp",
+    "service:drive": "yarn cozy-konnector-dev -m src/photos/targets/manifest.webapp",
+    "service:photos": "yarn cozy-konnector-dev -m src/photos/targets/manifest.webapp"
```

Comment:
since your commit is "adding tooling", can you explain how to use them within the commit message?

Reply:
Good point, I added a text in the commit message